contracts/src/StakeTable.sol

```diff
@@ -102,26 +116,68 @@ contract StakeTable is AbstractStakeTable {
     }
 
     /// @notice Get the next available epoch for new registration.
-    /// @return Number of the epoch when the user can register.
-    function nextRegistrationEpoch() external view override returns (uint64) {
-        // TODO implement queue logic
-        return currentEpoch() + 1;
+    /// @return Number of the epoch when the user can register and size of the queue in that epoch.
+    function nextRegistrationEpoch() external view override returns (uint64, uint64) {
```

Comment:
imo, we shouldn't return "the size after the user register" cuz this function should be totally oblivious of whether or when the caller will register, it could be people are just calling this function for fun/other purposes.

it should return the number of people already in the line!

Reply:
Yes, done in f30159c.